### PR TITLE
avoid extra lookup when `fastbootInfo` reference already exist

### PR DESCRIPTION
**Pull request by bekzod**
since `fastbootInfo` already instantiated and extra `lookup` does not give any value here

### CODE REVIEW

File: src/ember-app.js [hunk 1]
```diff
@@ -235,7 +239,7 @@ class EmberApp {
       })
       .then(() => result.instanceBooted = true)
       .then(() => instance.visit(path, bootOptions))
-      .then(() => waitForApp(instance))
+      .then(() => fastbootInfo.deferredPromise)
```
Comment (kratiahuja) [inline src/ember-app.js]: What if during the `visit`, the instance mutated the `fastbootInfo` object? Can you make sure this isn't a stale copy?

Comment (bekzod) [inline src/ember-app.js]: even if `fastbootInfo` is mutated it points to same object right, and it is registered in registry as `{ instantiate: false }` which means registry doesn't do anything special it comes out same as it comes in, correct if I am wrong :)

Comment (kratiahuja) [inline src/ember-app.js]: I am not worried about the registry. I am talking about the mutations during `visit` which is on the instance and not the application. If you can verify the objects are same via a simple test of changing the request property on the `fastbootInfo` in your app, this should be good.

Comment (bekzod) [inline src/ember-app.js]: added :)

**Pull request merged**